eth/vm/base.py

```diff
@@ -912,7 +928,14 @@ def validate_header(cls,
                 )
 
             if check_seal:
-                cls.validate_seal(header)
+                try:
+                    cls.validate_seal(header)
+                except ValidationError:
+                    cls.cls_logger.warning(
+                        "Failed to validate header proof of work on header: %r",
+                        header.as_dict()
```

Comment:
This is an example of somewhere that I'd like to use the built-in `RLP.Serializeable.__repr__`